### ISSUE TITLE
'Cannot read property 'charAt' of undefined'

### ISSUE THREAD

**Issue by lukaVarga** [EducationLink/vue-tel-input#124]
After updating to `v3.1.1` (from `v2.1.5`), I am getting the following error whenever I try to select a different country in the dropdown than the default one:

```
TypeError: Cannot read property 'charAt' of undefined
    at new Y (vendors.app.js:42908)
    at Y (vendors.app.js:42908)
    at VueComponent.choose (vendors.app.js:43404)
    at click (vendors.app.js:43730)
```

This happens only if I specify:
`:input-options="{ showDialCode: true }"` -> if the `showDialCode` is `false`, there is no error

It's happening within the `choose` method, when calling `PhoneNumber(this.phoneObject.number.national, this.activeCountry.iso2).getNumber('international')`

**Comment by iamstevendao** [comment]
Thanks for the report @lukaVarga. It's fixed in [v3.1.2](https://github.com/EducationLink/vue-tel-input/releases/tag/v3.1.2).

**Comment by lukaVarga** [comment]
@iamstevendao cheers, thanks for a quick response and a quick fix :) I can confirm it's working :)
